bindings/rust/s2n-tls-sys/templates/Cargo.template

```diff
@@ -27,6 +27,7 @@ default = []
 # preserve the cmake feature in case any consumers had it enabled before
 cmake = []
 quic = []
+fips = ["aws-lc-rs/fips"]
 pq = []
```

Comment:
Do we have to worry about openssl too, or is that not a concern for the bindings?

Reply:
Rust bindings only build with [aws-lc-rs](https://github.com/aws/s2n-tls/pull/4527/files#diff-02b6b35be0c00127505a1aa88bd203ab290e779306a0150398d0dc966302cc77L38)